app/assets/javascripts/components/GlobalNav.js

```diff
@@ -121,7 +285,15 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities'], function($
       .parents('[data-dough-nav-clumps]').toggleClass('is-active');
   };
 
-  GlobalNav.prototype._openDesktopSubNav = function(index) {
+  GlobalNav.prototype._openDesktopSubNav = function(index, delay) {
+    var timeoutDelay;
+
+    if (delay) {
+      timeoutDelay = this.delay;
+    } else {
+      timeoutDelay = 0;
+    }
```

Comment:
I think you can condense these 7 lines to simply `var timeoutDelay = this.delay || 0'`.

If we were using ES2015, this would be a great use case for [default parameters](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters).